packages/appeals-service-api/src/services/back-office-v2/index.js

```diff
@@ -72,25 +73,48 @@ class BackOfficeV2Service {
 		if (!isValidAppealTypeCode(appellantSubmission.appealTypeCode))
 			throw new Error(`Appeal submission ${appellantSubmissionId} has an invalid appealTypeCode`);
 
+		logger.info(
+			`mapping appeal ${appellantSubmission.appealId} to ${appellantSubmission.appealTypeCode} schema`
+		);
 		const mappedData = await formatters.appeal[
 			appealTypeCodeToAppealId[appellantSubmission.appealTypeCode]
 		](appellantSubmission);
+		logger.debug({ mappedData }, 'mapped appeal');
 
+		logger.info(`validating appeal ${appellantSubmission.appealId} schema`);
 		/** @type {Validate<AppellantSubmissionCommand>} */
 		const validator = getValidator('appellant-submission');
 		if (!validator(mappedData)) {
 			throw new Error(
-				`Payload was invalid wen checked against appellant submission command schema`
+				`Payload was invalid when checked against appellant submission command schema`
 			);
 		}
 
+		logger.info(`forwarding appeal ${appellantSubmission.appealId} to service bus`);
 		const result = await forwarders.appeal([mappedData]);
 
 		await markAppealAsSubmitted(appellantSubmission.id);
 
-		await sendSubmissionConfirmationEmailToAppellantV2(appellantSubmission, email);
+		logger.info(`sending appeal submitted emails for ${appellantSubmission.appealId}`);
+
+		let emailFailed = false;
+		try {
+			await sendSubmissionConfirmationEmailToAppellantV2(appellantSubmission, email);
+		} catch (err) {
+			logger.error({ err }, 'failed to sendSubmissionConfirmationEmailToAppellantV2');
+			emailFailed = true;
+		}
+
+		try {
+			await sendSubmissionReceivedEmailToLpaV2(appellantSubmission, email);
+		} catch (err) {
+			logger.error({ err }, 'failed to sendSubmissionConfirmationEmailToAppellantV2');
+			emailFailed = true;
+		}
 
-		await sendSubmissionReceivedEmailToLpaV2(appellantSubmission, email);
+		if (emailFailed) {
+			throw new Error('failed to send submission email');
```

Comment:
Is there any value in having a different/expanded emailFailed for lpa and appellant in order to make it easier to identify which failed?

Reply:
I was going to say we'd have the logger.error but I used the wrong function name